admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -59,7 +59,7 @@ $(document).ready(function() {
   /* validate price fields , as Thomas de Nabord said */
   $(".money-type input[type='text']").change(function validate(event) {
     var inputValue = $(this).val();
-    var parsedValue = truncateDecimals(parseFloat(inputValue),6);
+    var parsedValue = (new Number(inputValue)).toFixed(6);
```

Comment:
You mean in the `return` step ?

I would not agree, this add too many zeros if I do not reach this precision (which becomes difficult to read in a small input).